update_proto.py

```diff
@@ -0,0 +1,36 @@
+import os
+import pathlib
+
+import requests
+from grpc_tools.protoc import main as grpc_tools_protoc_main
+
+zeebe_proto_version = "8.6.6"
+
+
+def generate_proto():
+    proto_dir = pathlib.Path("pyzeebe/proto")
+    proto_file = proto_dir / "gateway.proto"
+    for path in proto_dir.glob("*pb2*"):
+        os.remove(path)
+
+    proto_url = f"https://raw.githubusercontent.com/camunda/camunda/refs/tags/{zeebe_proto_version}/zeebe/gateway-protocol/src/main/proto/gateway.proto"
+    proto_content = requests.get(proto_url, allow_redirects=True)
+    with proto_file.open("wb") as tmpfile:
+        tmpfile.write(proto_content.content)
+
+        grpc_tools_protoc_main(
+            [
+                "--proto_path=.",
+                "--python_out=.",
+                "--mypy_out=.",
+                "--grpc_python_out=.",
+                "--mypy_grpc_out=.",
+                os.path.relpath(tmpfile.name),
+            ]
+        )
+
+    proto_file.unlink()
+
+
+if __name__ == "__main__":
+    generate_proto()
```

Comment:
Add some error checking otherwise, previous version in cache is used to generate protocol buffers.
Also command line usage is available.

Maybe we can integrate this with the pyproject.toml ?

Reply:
> Maybe we can integrate this with the pyproject.toml ?

This script is used only in development time, so I think it isn't necessary.

Reply:
✅ Good

Btw. got a little bit confused while looking at the latest changes and in the discussion (Suggestion is outdated, but the commited changes are the same).